docs/reference/ccr/use-cases.asciidoc

```diff
@@ -0,0 +1,121 @@
+or unavailable.
+
+In the following diagram, data from Datacenter A is replicated to
+Datacenter B and Datacenter C, which both have a read-only copy of the leader
+index from Datacenter A.
+
+image::images/ccr-arch-multiple-dcs.png[Production datacenter that replicates data to two other datacenters]
+
+[discrete]
+[[ccr-chained-replication]]
+===== Chained replication
+You can replicate data across multiple datacenters to form a replication
+chain. In the following diagram, Datacenter A contains the leader index.
+Datacenter B replicates data from Datacenter A, and Datacenter C replicates
+from the follower indices in Datacenter B. The connection between these
+datacenters forms a chained replication pattern.
+
+image::images/ccr-arch-chain-dcs.png[Three datacenters connected to form a replication chain]
+
+[discrete]
+[[ccr-bi-directional-replication]]
+===== Bi-directional replication
+In a https://www.elastic.co/blog/bi-directional-replication-with-elasticsearch-cross-cluster-replication-ccr[bi-directional replication] setup, all clusters have access to view
+all data, and all clusters have an index to write to without manually
+implementing failover. Applications can write to the local index within each
+datacenter, and read across multiple indices for a global view of all
+information.
+
+This configuration requires no manual intervention when a cluster or datacenter
+is unavailable. In the following diagram, if Datacenter A is unavailable, you can continue using Datacenter B without manual failover. When Datacenter A
+comes online, replication resumes between the clusters.
+
+image::images/ccr-arch-bi-directional.png[Bi-directional configuration where each cluster contains both a leader index and follower indices]
+
+NOTE: This configuration is useful for index-only workloads, where no updates
+to document values occur. In this configuration, documents indexed by {es} are
+immutable. Clients are located in each datacenter alongside the {es}
+cluster, and do not communicate with clusters in different datacenters.
+
+[[ccr-data-locality]]
+==== Data locality
+Bringing data closer to your users or application server can reduce latency
+and response time. This methodology also applies when replicating data in {es}.
+For example, you can replicate a product catalog or reference dataset to 20 or
+more datacenters around the world to minimize the distance between the data and
+the application server.
+
+In the following diagram, data is replicated from one datacenter to three
+additional datacenters, each in their own region. The central datacenter
+contains the leader index, and the additional datacenters contain follower
+indices that replicate data in that particular region. This configuration
+puts data closer to the application accessing it.
+
+image::images/ccr-arch-data-locality.png[A centralized datacenter replicated across three other datacenters, each in their own region]
```

Comment:
This reverses the data flow from the earlier diagrams--I'd be consistent.

Reply:
Good point. I'll reverse the diagram so that data flows left -> right.